charts/illumidesk/values.yaml

```diff
@@ -30,6 +30,13 @@ jupyterhub:
   # JupyterHub's application settings
   # ref: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/jupyterhub/values.yaml
   hub:
+    # Jupyterhub resources regarding requests for CPU and memory
+    resources:
+      requests:
+        # CPU that is allocated to the hub
+        cpu: "200m"
+        # memory that is allocated to the hub
+        memory: "400Mi"
```

Comment:
@Abhi94N shouldn't we include the option for limits too?

Reply:
I don't believe there are limit options for that deployment